src/net/udp.rs

```diff
@@ -150,7 +150,6 @@ impl UdpSocket {
     // so simply disable the test on FreeBSD.
     #[cfg_attr(not(target_os = "freebsd"), doc = " ```")]
     #[cfg_attr(target_os = "freebsd", doc = " ```no_run")]
-    /// ```
```

Comment:
I'm a bit confused (I'm not used to doc examples guarded by cfg_attr).

Are the triple ticks not needed here? Is it because of the cfg_attr? Huh... who knew.

Reply:
I was surprised as well to see this, but if you take a look at the [local_addr](https://carllerche.github.io/mio/mio/net/struct.UdpSocket.html#method.local_addr) function docs, removing the triple ticks fixes the issue.